integration/hibernate-5.4/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2014 - 2018 Blazebit.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.blazebit</groupId>
+        <artifactId>blaze-persistence-integration</artifactId>
+        <version>1.3.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>blaze-persistence-integration-hibernate-5.4</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Blazebit Persistence Integration Hibernate 5.4</name>
```

Comment:
Could you please also add the automatic module name property?

Reply:
You mean https://github.com/Blazebit/blaze-persistence/blob/master/integration/hibernate-5.3/pom.xml#L34 ? I suppose I'll also have to add https://github.com/Blazebit/blaze-persistence/blob/master/integration/hibernate-5.3/pom.xml#L103-L132 then. This would by the way point to the 53 classes, as these aren't renamed by the shading. Is that sensible for the 5.4 integration artifact?

Reply:
Yep, both of these sections are necessary. There is only one integration supposed to be used, so it's ok to have the same definition.